test/test_config.rb

```diff
@@ -160,7 +160,8 @@ def test_inline
     prepare_config
     opts = {
       :config_path => "#{TMP_DIR}/config_test_1.conf",
-      :inline_config => "<source>\n  type http\n  port 2222\n </source>"
+      :inline_config => "<source>\n  type http\n  port 2222\n </source>",
+      :use_v1_config => false
```

Comment:
Sorry, I can't understand why adding log rotation setting affects to this even I read the commit message.
Could you describe more detail?

Reply:
I've added more explanation in the commit message.